server/controllers/request.controller.js

```diff
@@ -0,0 +1,61 @@
+const RequestInfo = require('../models/request.model.js');
+
+exports.getRequests = async (req, res) => {
+    try {
+        const requestInfos = await RequestInfo.find();
+        res.status(200).json(requestInfos);
+    } catch (error) {
+        res.status(404).json( {message: error.message} );
+    }
+}
+
+exports.getById = (getRequestById, (req, res) => {
+    res.json(res.requests);
+}) 
+
+exports.createRequest = async (req, res) => {
+    const newRequest = new RequestInfo({
+        user_id: req.body.user_id,
+        sitter_id: req.body.sitter_id,
+        start: req.body.start,
+        end: req.body.end
+    });
+
+    try {
+        await newRequest.save();
+        res.status(201).json(newRequest);
+    } catch (error) {
+        res.status(409).json( { message: error.message } );
```

Comment:
Again, please be mindful of the status codes we are using. [409 Conflict](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409) isn't really a good fit for this kind of response, make sure to understand the description of the status codes so we are following REST convention.